src/control/server/mgmt_system.go

```diff
@@ -982,10 +980,6 @@ func (svc *mgmtSvc) SystemStop(ctx context.Context, req *mgmtpb.SystemStopReq) (
 	// First phase: Prepare the ranks for shutdown, but only if the request is for an unforced
 	// full system stop.
 	if !fReq.Force {
-		if !fReq.FullSystem {
```

Comment:
The initial reason for the restriction was to maintain parity for the legacy behaviour. The prep-shutdown stage was initially intended to prevent the proliferation of exclusion events during a full system shutdown. Allowing calling of `ds_pool_disable_exclude` for system stop on a subset of ranks is unlikely to cause other unintended behaviour but it's not impossible, just fyi.

Reply:
Good to know. Thanks.